bats/helpers/ln.bash

```diff
@@ -1,6 +1,120 @@
 CURRENT_FILE=${BASH_SOURCE:-bats/helpers/.}
 source "$(dirname "$CURRENT_FILE")/_common.bash"
 source "$(dirname "$CURRENT_FILE")/cli.bash"
+source "$(dirname "$CURRENT_FILE")/subscriber.bash"
+
+fund_user_lightning() {
+  local token_name=$1
+  local wallet_id_name="$token_name.${2}_id" # btc_wallet or usd_wallet
+  local amount=$3
+
+  variables=$(
+    jq -n \
+    --arg wallet_id "$(read_value $wallet_id_name)" \
+    '{input: {walletId: $wallet_id}}'
+  )
+  exec_graphql "$token_name" 'ln-no-amount-invoice-create' "$variables"
+  invoice="$(graphql_output '.data.lnNoAmountInvoiceCreate.invoice')"
+
+  payment_request="$(echo $invoice | jq -r '.paymentRequest')"
+  [[ "${payment_request}" != "null" ]]
+  payment_hash="$(echo $invoice | jq -r '.paymentHash')"
+  [[ "${payment_hash}" != "null" ]]
+
+  lnd_outside_cli payinvoice -f \
+    --pay_req "$payment_request" \
+    --amt "$amount"
+
+  retry 15 1 check_for_ln_initiated_settled "$token_name" "$payment_hash"
+}
+
+lnds_init() {
```

Comment:
This should probably be a setup step in the tilt file.

Reply:
Added the file here #3718